cypress/e2e/dashboard.cy.ts

```diff
@@ -198,4 +198,250 @@ describe("Dashboard", () => {
+          .should("not.be.disabled");
+      });
+    });
+
+    describe("Payment modal", () => {
+      beforeEach(() => {
+        login(keys.loggedUserFundedSecretKey);
+        cy.get('[data-cy="send-payment-button"]')
+          .should("exist")
+          .trigger("click");
+      });
+
+      it("Should display the payment modal when the user clicks on the send assets button", () => {
+        login(keys.loggedUserFundedSecretKey);
+        cy.get('[data-cy="send-payment-button"]')
+          .should("exist")
+          .trigger("click");
+        cy.get('[data-cy="payment-modal"]').should("exist");
+      });
+
+      it("Should have a close button with the text 'Close'", () => {
+        cy.get('[data-cy="close-payment-modal-button"]')
+          .should("exist")
+          .should("have.text", "Close");
+      });
+
+      it("Should be unmounted when the user clicks on the close button", () => {
+        cy.get('[data-cy="close-payment-modal-button"]').trigger("click");
+        cy.get('[data-cy="payment-modal"]').should("not.exist");
+      });
+
+      it("Should have a send button with the text 'Send'", () => {
+        cy.get('[data-cy="send-payment-modal-button"]')
+          .should("exist")
+          .should("have.value", "Send");
+      });
+
+      it("Should have a source account input field with the 'Signer' label", () => {
+        cy.get('[data-cy="signer-account-input"]').should("exist");
+        cy.get('[data-cy="signer-account-label"]')
+          .should("exist")
+          .should("have.text", "Signer");
+      });
+
+      it("Should have a destination account input field with the 'Destination' label", () => {
+        cy.get('[data-cy="destination-account-input"]').should("exist");
+        cy.get('[data-cy="destination-account-label"]')
+          .should("exist")
+          .should("have.text", "Destination");
+      });
+
+      it("Should have an amount input field with the 'Amount' label", () => {
+        cy.get('[data-cy="amount-input"]').should("exist");
+        cy.get('[data-cy="amount-label"]')
+          .should("exist")
+          .should("have.text", "Amount");
+      });
+
+      it("Should have a memo input field with the 'Memo' label", () => {
+        cy.get('[data-cy="memo-input"]').should("exist");
+        cy.get('[data-cy="memo-label"]')
+          .should("exist")
+          .should("have.text", "Memo");
+      });
+
+      it("Should have a time out input field with the 'Timeout' label", () => {
+        cy.get('[data-cy="time-out-input"]').should("exist");
+        cy.get('[data-cy="time-out-label"]')
+          .should("exist")
+          .should("have.text", "Timeout");
+      });
+
+      it("Should have a 'Send Lumens' title", () => {
+        cy.get('[data-cy="send-payment-modal-title"]')
+          .should("exist")
+          .should("have.text", "Send Lumens");
+      });
+    });
+
+    describe("Send assets funcionality", () => {
+      const testingPaymentData = {
```

Comment:
Avoid using the word `data`, please.

Reply:
Also, you can destructure the properties needed here.

Reply:
I have removed de testingPaymentData name and destructured the properties.